src/utils/parseMissingQueryParams.ts

```diff
@@ -1,18 +1,27 @@
-export const parseMissingQueryParams = (queryStringParameters: any) => {
+/**
+ * AWS lambda returns event.queryStringParameters containing only string values.
+ * If you pass null, undefined or 47 in event.queryStringParameters
+ * they will be received as "null", "undefined" and "47"
+ *
+ * This method is required to cast all the parameters to the correct type
+ * @param queryStringParameters
+ * @param numericParameters List of the queryString parameter names to be casted to number
+ */
+export const parseAndCastQueryParams = (queryStringParameters: any, numericParameters: string[]) => {
     const queryParams = Object.assign({}, queryStringParameters);
 
-    if (queryParams.vehicleAxles === "null") {
-        queryParams.vehicleAxles = null;
-    } else if ( queryParams.vehicleAxles !== undefined ) {
-        queryParams.vehicleAxles = parseInt(queryParams.vehicleAxles, 10);
-    }
+    Object.keys(queryParams).forEach((parameterName: string) => {
+        if (queryParams[parameterName] === "null" || queryParams[parameterName] === "undefined") {
+            queryParams[parameterName] = null;
+        }
+        if (numericParameters.includes(parameterName)) {
+            queryParams[parameterName] = castToNumber(queryParams[parameterName]);
+        }
+    });
 
-    if (queryParams.vehicleSize === "undefined") {
-        queryParams.vehicleSize = null;
-    }
-
-    if (queryParams.vehicleConfiguration === "null") {
-        queryParams.vehicleConfiguration = null;
-    }
     return queryParams;
 };
+
+function castToNumber(parameter: string): number {
+    if (parameter === null) { return parameter; } else { return parseInt(parameter, 10); }
```

Comment:
Could be simplified to
`return parameter ? null : parseInt(parameter, 10)`

Reply:
You're right, doing it

Reply:
Done